facia-press/app/frontpress/jsonModels.scala

```diff
@@ -113,7 +117,11 @@ case class ItemMeta(
   showKickerSection: Option[JsValue],
   showMainVideo: Option[JsValue],
   imageSlideshowReplace: Option[Boolean],
-  slideshow: Option[JsValue]
+  slideshow: Option[JsValue],
+  imageReplace: Option[Boolean],
+  imageSrc:       Option[JsValue],
+  imageSrcWidth:  Option[JsValue],
+  imageSrcHeight: Option[JsValue]
```

Comment:
Bit worried about this, I thought case classes had a parameter limit? Did it magically disappear? Works on my machine fine.